dts/arm/nordic/nrf54l15_cpuapp.dtsi

```diff
@@ -33,6 +33,14 @@
 		};
 	};
 
+	clocks {
+		lfxo: lfxo {
+			compatible = "nordic,nrf-lfxo";
+			#clock-cells = <0>;
+			clock-frequency = <32768>;
```

Comment:
Isn't this superfluous since
```
  clock-frequency:
    const: 32768
```
?

Reply:
it's still required

```
devicetree error: 'clock-frequency' is marked as required in 'properties:' in /home/gmarull/ws/nordic/zephyrproject/zephyr/dts/bindings/clock/nordic,nrf-lfxo.yaml, but does not appear in <Node /clocks/lfxo in '/home/gmarull/ws/nordic/zephyrproject/zephyr/misc/empty_file.c'>
```

Reply:
`# 'const' is just a special case of an enum with a single possible value` I had no idea, the more you know